packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
-                  color: "transparent",
+  return property ? (
+    <>
+      <TableRow>
+        <TableCell>
+          <Typography variant="smallTextLabels" fontWeight={500}>
+            {property.title}
+          </Typography>
+        </TableCell>
+        <TableCell>
+          <PropertyExpectedValues property={property} />
+        </TableCell>
+        <TableCell
+          ref={(ref: HTMLDivElement) => setAnchorEl(ref)}
+          sx={({ palette, transitions }) => ({
+            px: "0px !important",
+            position: "relative",
+            cursor: "pointer",
+            textAlign: "center",
+            transition: transitions.create("border-color"),
+            border: `1px solid ${
+              multipleValuesMenuOpen ? palette.gray[40] : "transparent"
+            } !important`,
+            "&:hover": {
+              borderColor: `${palette.gray[40]} !important`,
+            },
+          })}
+          onClick={() => setMultipleValuesMenuOpen(true)}
+        >
+          <Box
+            sx={({ palette, transitions }) => ({
+              display: "inline-flex",
+              borderRadius: "4px 30px 30px 4px",
+              backgroundColor: "transparent",
+              transition: transitions.create(["padding", "background-color"]),
+              ...(array && !multipleValuesMenuOpen
+                ? {
+                    py: 0.5,
+                    px: 0.75,
+                    background: palette.gray[20],
+                  }
+                : {}),
+            })}
+          >
+            <Checkbox
+              sx={{
+                zIndex: 1,
+              }}
+              checked={array}
+              onChange={(event) => {
+                update(propertyIndex, {
+                  ...draftProperty,
+                  array: event.target.checked,
+                  minValue: minValue ?? 0,
+                  maxValue: maxValue ?? 0,
+                });
+              }}
+            />
+            <Collapse
+              orientation="horizontal"
+              in={array && !multipleValuesMenuOpen}
+            >
+              <Typography
+                variant="smallTextLabels"
+                sx={{
+                  display: "flex",
+                  ml: 1,
+                  fontWeight: 500,
+                  whiteSpace: "nowrap",
+                  color: ({ palette }) => palette.gray[70],
+                }}
+              >
+                {minValue !== maxValue
+                  ? `${minValue} to ${maxValue}`
+                  : minValue}
+              </Typography>
+            </Collapse>
+          </Box>
+        </TableCell>
+        <CenteredTableCell sx={{ textAlign: "center" }}>
+          <Checkbox
+            checked={required}
+            onChange={(event) => {
+              update(propertyIndex, {
+                ...draftProperty,
+                required: event.target.checked,
+              });
+            }}
```

Comment:
This should use `register`

Reply:
Using `register` since 669f2a5700d2e0a6385302b661eb077a24186144.